benchmarks/brute_force_vs_bvh/brute_force_vs_bvh_timpl.hpp

```diff
@@ -90,9 +90,8 @@ static void run_fp(int nprimitives, int nqueries, int nrepeats)
     {
       Kokkos::Timer timer;
       ArborX::BoundingVolumeHierarchy<MemorySpace,
-                                      ArborX::Details::PairIndexVolume<Point>>
-          bvh{space, ArborX::Details::LegacyValues<decltype(primitives), Point>{
-                         primitives}};
+                                      ArborX::PairValueIndex<Point>>
+          bvh{space, ArborX::AttachIndices<decltype(primitives)>{primitives}};
```

Comment:
Any reason why you didn't rely on CTAD?
```suggestion
          bvh{space, ArborX::AttachIndices{primitives}};
```
same comment everywhere

Reply:
Yes, annoyingly CTAD fails for CUDA 11.0.3.